### ISSUE TITLE
Multiline tablecell

### ISSUE THREAD

**Issue by mchusovlianov** [unidoc/unidoc#166]
Is there a way how to create table cell with multiline text?

**Comment by gunnsth** [comment]
@mchusovlianov Can you please try the fix provided in branch creator-tablecell-wrapping-fix ?
For the paragraph content to wrap, you need to set p.SetEnableWrap(true)

**Comment by mchusovlianov** [comment]
@gunnsth I've checked creator-tablecell-wrapping-fix and it doesn't look working. This is an example how we try to use it:
```
c := creator.New()
issuerTable := creator.NewTable(2)

p := creator.NewParagraph("Line1")
p.SetFontSize(10)
p.SetColor(creator.ColorWhite)

cell := issuerTable.NewCell()
cell.SetBorder(creator.CellBorderStyleBox, 1)
cell.SetContent(p)

p = creator.NewParagraph("Line1\r\nLine2")
p.SetFontSize(10)
p.SetColor(creator.ColorWhite)
p.SetEnableWrap(true)

cell = issuerTable.NewCell()
cell.SetBorder(creator.CellBorderStyleBox, 1)
cell.SetContent(p)

c.Draw(issuerTable)
```

**Comment by gunnsth** [comment]
Ah I see.  The Paragraph wrapping will wrap the Paragraph text automatically if it flows out of bounds, so it is not the same as the newline wrapping.

Will look into what is needed to get newline wrapping working as well.

**Comment by mchusovlianov** [comment]
@gunnsth That works! Thank a lot!

**Comment by gunnsth** [comment]
Closed by merge of PR #179 into master (and release of 2.1.1).
